catprinter/img.py

```diff
@@ -120,19 +120,32 @@ def read_img(
     filename,
     print_width,
     img_binarization_algo,
+    no_resize=False
 ):
+
     im = cv2.imread(filename, cv2.IMREAD_GRAYSCALE)
     height = im.shape[0]
     width = im.shape[1]
-    factor = print_width / width
-    resized = cv2.resize(
-        im,
-        (
-            print_width,
-            int(height * factor)
-        ),
-        interpolation=cv2.INTER_AREA)
 
+    if not no_resize:
```

Comment:
Nit: I have some trouble reading double negatives like this. Maybe we can invert the if-else and possibly some nesting with something like:

```
if no_resize and width <= print_width: # Print as is.
  ...
else: # Resize to fit width. Preserves aspect ratio.
  ...
```

What do you think?